### DISCUSSION TITLE
QR wg easy

### DISCUSSION THREAD

**Discussion by Selys-u** [xtrime-ru/antizapret-vpn-docker#118] [Q&A]
После установки связки anti + wg-easy + adguard, при создании конфига в wg-easy невозможно считать qr на клиентах. Выдаёт ошибку

Можно как-то пофиксить?

**Comment by xtrime-ru** [comment]
Проверил - проблема не воспроизводится. Точно используется wg-easy a не amnezia-wg-easy? 
Что за клиент и какая OS? Файлы конфигурации работают?
